pynsee/localdata/_find_latest_local_dataset.py

```diff
@@ -61,6 +64,9 @@ def _find_latest_local_dataset(dataset_version, variables, update):
                 update=True
             )
         else:
-            print("Latest dataset version previously found has been used\nSet update=True to get the most up-to-date data")
+            logger.info(
```

Comment:
It's probably worth discussing what should be the visibility of this message...
cc @hadrilec

Reply:
Yes, this is the case where this should be discussed. In my opinion, this behaviour is in accordance with documentation and shouldn't result in a warning (cf. supra about the guidelines).

Reply:
I would tend to agree